devlab/tests/base.py

```diff
@@ -31,6 +32,36 @@
 OPENSTACK_RELEASES = {'192.168.1.2': 'grizzly',
                       '192.168.1.3': 'icehouse',
                       '192.168.1.8': 'juno'}
+SWIFT_AUTH_VERSION = '2'
+
+
+class SwiftConnection(object):
+    def __init__(self, auth_url, username, password, tenant):
+        self.auth_url = auth_url
+        self.username = username
+        self.password = password
+        self.tenant = tenant
+
+    def __enter__(self):
+        class Swift(object):
```

Comment:
I don't see ether, lets just add swift client object initialization to BasePrerequisites object __init__ method as we done that for other clients